rig/routing_table/utils.py

```diff
@@ -0,0 +1,238 @@
+import warnings
+from rig.routing_table import RoutingTableEntry
+
+
+def table_is_subset_of(entries_a, entries_b):
+    """Check that every key matched by every entry in one table results in the
+    same route when checked against the other table.
```

Comment:
Possibly, but I quite like having it handy for checking how well (or otherwise) I'm driving Espresso. I appreciate that's probably a bad reason for having it hang around though.